File: 8-kyu/how-old-will-i-be-in-2099/how-old-will-i-be-in-2099.go
package kata
​
import "fmt"
​
func calculateAge(born, year int) string {
  age := year - born
  if age > 1 {
    return fmt.Sprintf("You are %d years old.", age)
  } else if age < -1 {
    return fmt.Sprintf("You will be born in %d years.", -age)
  } else if age == 1 {
    return "You are 1 year old."
  } else if age == -1 {
    return "You will be born in 1 year."
  }
  return "You were born this very year!"// enter your code here.
}
​

File: 8-kyu/how-old-will-i-be-in-2099/how-old-will-i-be-in-2099.py
def calculate_age(year_of_birth, current_year):
    age = current_year - year_of_birth
    if age > 1:
        return f'You are {age} years old.'
    elif age < -1:
        return f'You will be born in {-age} years.'
    elif age == 1:
        return 'You are 1 year old.'
    elif age == -1:
        return 'You will be born in 1 year.'
    else:
        return 'You were born this very year!'